lib/internal/bootstrap_node.js

```diff
@@ -344,11 +344,7 @@
                                              wrappedConsole[key],
                                              config);
     }
-    for (const key of Object.keys(originalConsole)) {
-      if (wrappedConsole.hasOwnProperty(key))
-        continue;
-      wrappedConsole[key] = originalConsole[key];
-    }
+    wrappedConsole.inspector = originalConsole;
```

Comment:
I would just completely ignore the dummy inspector methods, i.e. do not expose them at all.

Reply:
If the inspector methods are not provided, users don't have a way of sending any of the messages that aren't also provided by `Console.prototype` to the inspector console.

I had originally thought of exposing something like  `require('inspector').console`, but I'm not sure that's something I can set up at `bootstrap_node.js` time, so `console.inspector` seemed to be a good second option.

Reply:
@bengl I understand that implication. However, I feel it's ugly from a user experience standpoint to require users to use `console.inspector ? console.inspector.table || console.table` in polymorphic code: either it should work the way it does everywhere, or it shouldn't.

Also, these console methods only became exposed in 8.0.0, so there isn't much compatibility concern.

Reply:
For certain functions that inherently do not deal with stdio, like `console.markTimeline`, `console.profile`, `console.profileEnd`, `console.timeStamp`, `console.timeline`, and `console.timelineEnd`, I think we should just keep them as they are and maybe document them. There isn't any work needed by Node.js' Console to "support" those methods because they are no-ops in the console anyway.

That leaves `console.debug`, `console.dirxml`, and `console.table` truly unimplemented Console methods.

Reply:
@TimothyGu Is that a "-1" opposition or more a "-0"?